app/controllers/bgeigie_imports_controller.rb

```diff
@@ -95,6 +95,15 @@ def destroy
     end
   end
 
+  def kml
+    bgeigie_import = BgeigieImport.find(params[:id])
+    ::Actions::BgeigieImports::Kml.new(
```

Comment:
_that_ makes sense. `render 'beigie_logs', formats: [:kml], content_type: Mime::KML` is what I would have expected, but if it doesn't like the kml then I'm cool with this and we can move on.

Maybe add a comment or gh issue to revisit after rails 4?